CHANGELOG.md

```
@@ -1,6 +1,8 @@
-# 7.1.1 (in progress)
+# 7.2.0 (in progress)
 
-* Your contribution here!
+##### New Features
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h5

(MD001, heading-increment)